### PR TITLE
Backprop more reps

### PR DESCRIPTION
**Pull request by antalszava**

### CODE REVIEW

File: demonstrations/tutorial_backprop.py [hunk 1]
```diff
@@ -180,8 +180,8 @@ def circuit(params):
 
 import timeit
 
-reps = 3
-num = 10
+reps = 5
```
Comment (antalszava) [inline demonstrations/tutorial_backprop.py:183]:
```suggestion
reps = 10
```

File: demonstrations/tutorial_backprop.py [hunk 2]
```diff
@@ -276,8 +276,8 @@ def circuit(params):
...
```
Comment (antalszava) [inline demonstrations/tutorial_backprop.py:279]:
```suggestion
reps = 10
```

**Comment by josh146** [comment]
@antalszava curious the reason for this change?

**Comment by antalszava** [comment]
@josh146 the two diagrams seem to be less accurate than before, quite likely due to the change to use `autograd` as per #256.

Nonetheless, the outputs are correct. This PR is just an attempt to make the outputs more accurate.

**stable**
image

image

**latest** (using `dev` without no changes)
image

image

**Increased reps and nums** (updating `dev` with this PR)
image

image

**Comment by josh146** [comment]
@antalszava, thanks for investigating this!

I have to admit, however, that 

looks relatively okay to me --- it _is_ a bit 'messy', but the trend line is still clear.

With this change, the demo now takes 10 minutes to run rather than 4. Should we keep it as is to preserve the shorter running time?

**Comment by antalszava** [comment]
@josh146 definitely! Happy to keep it as is :slightly_smiling_face: Will close this.

**Pull request closed without merging**